src/tools/ceph_objectstore_tool.cc

```diff
@@ -1909,10 +1909,10 @@ int main(int argc, char **argv)
     ("journal-path", po::value<string>(&jpath),
      "path to journal, mandatory for filestore type")
     ("pgid", po::value<string>(&pgidstr),
-     "PG id, mandatory for info, log, remove, export, rm-past-intervals")
+     "PG id, mandatory for info, log, remove, export, rm-past-intervals, mark-complete")
```

Comment:
You also need to check that mark-complete is specified with a pgid in the test at 2287.